exporter/clickhouseexporter/factory.go

```diff
@@ -38,12 +38,13 @@ func createDefaultConfig() component.Config {
 		QueueSettings:    queueSettings,
 		BackOffConfig:    configretry.NewDefaultBackOffConfig(),
 		ConnectionParams: map[string]string{},
-		Database:         defaultDatabase,
+		Database:         "default",
```

Comment:
The const `defaultDatabase` still exists in the code with a value of `"default"`, can we use that instead?

Reply:
I originally avoided this because I didn't like using the const from `config.go`. I have applied this change for now just so we can get this merged asap.